File: Eclipse 2/codeLM starter-code/Java/NewWaveHoldem.java
import java.util.Scanner;

class NewWaveHoldem {
    static String newWaveHoldem(String[] hand, String[] communityCards) {
        // Your code here
    }

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int handLength = Integer.parseInt(s.nextLine());
        String[] hand = new String[handLength];
        for (int i = 0; i < handLength; i++) {
            hand[i] = s.nextLine();
        }
        int communityCardsLength = Integer.parseInt(s.nextLine());
        String[] communityCards = new String[communityCardsLength];
        for (int i = 0; i < communityCardsLength; i++) {
            communityCards[i] = s.nextLine();
        }

        String functionCallResult = newWaveHoldem(hand, communityCards);
        System.out.println(functionCallResult);
    }
}

File: Eclipse 2/codeLM starter-code/Python/new_wave_holdem.py
def new_wave_holdem(hand, community_cards):
  # Your code here


if __name__ == '__main__':
  hand_length = int(input())
  hand = []
  for _ in range(hand_length):
    hand.append(input())
  community_cards_length = int(input())
  community_cards = []
  for _ in range(community_cards_length):
    community_cards.append(input())

  function_call_result = new_wave_holdem(hand, community_cards)
  print(function_call_result)
